adafruit_max31856.py

```diff
@@ -45,7 +45,7 @@
 except ImportError:
     from ustruct import unpack
 
-__version__ = "0.0.0+auto.0"
+__version__ = "0.11.7"
```

Comment:
This needs to stay as `"0.0.0+auto.0"` There is another tool within our release infrastructure that fills this in for libraries that are downloaded from the bundle.